### DISCUSSION TITLE
Copilot4Eclipse Chat - 404 Not Found

### DISCUSSION THREAD

**Discussion by htfv** [Genuitec/Copilot4Eclipse-community#16] [Help] [answered]
I have a problem with the Copilot4Eclipse Chat showing the 404 Not Found error. I noticed the following messages in the ErrorLog:

```
Jetty server started: true uri: http://10.176.**.***:50218/chat
Chat App Path: http://localhost:50218/chat/chat.html?chatSessionId=17262263*****
```

As far as I unterstand, the problem is that Jetty listens on the external IP. Is there a way to force Jetty to listen on localhost?

Eclipse: 2024-09 RCP
Copilot4Eclipse: 1.2.1.202406240349

**Comment by wayneparrott** [comment]
Hi. Thanks for sharing.
Would you consider testing a new build of Copilot4Eclipse on your network to see if it resolves this issue? If yes please send email to support@genuitec.com  to my attention and include a link to this thread.
W

**Comment by htfv** [comment]
Yes, sure, I sent an e-mail.

**Comment by wayneparrott** [comment]
Please see your email for my follow up.
W

**Comment by dex-ds** [comment]
Hi, I run into the same issue. What was the solution? Is it a secret? ;-) cheers Daniel

Windows 11 Enterprise, Eclipse SDK 4.33, java 21, copilot4eclipse 1.2.2.202409201559

**Comment by wayneparrott** [comment]
@dex-ds  The 404 issue some ppl have experienced should be fixed in the upcoming 1.3 release. 
Would you be interested in trying a developer build to see if the issue is resolved for your environment?
If yes, send your details to support@genuitec.com, attn: Wayne and a link to this thread. 

Wayne
Copilot4Eclipse Team

**Comment by dex-ds** [comment, marked as answer]
I can confirm that the issue has been fixed with 1.3. Thank you
